source/scenario/mission3task1.vwf.yaml

```diff
@@ -123,7 +122,7 @@ children:
           - playSound:
             - musicSuccessShort
 
-        scenarioEnded:
+        scenarioEnded31:
```

Comment:
I've been using _3_1, in case we want to stick more numbers or letters in there later.